td3_trainer:
  ros__parameters:
    use_sim_time: true
    seed: 42
    eval_freq: 5000
    max_ep: 250
    eval_ep: 10
    max_timesteps: 1000000
    expl_noise: 1.0
    expl_decay_steps: 500000
    expl_min: 0.1
    batch_size: 64
    discount: 0.99999
    tau: 0.005
    policy_noise: 0.2
    noise_clip: 0.5
    policy_freq: 2
    buffer_size: 1000000
    file_name: "TD3_Turtlebot"
    save_model: true
    load_model: false
    random_near_obstacle: true
    environment_dim: 20

td3_tester:
  ros__parameters:
    use_sim_time: true
    seed: 42
    max_ep: 250
    file_name: "TD3_Turtlebot"
    environment_dim: 20
